docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.1'
+
+services:
+
+  mongo:
+    image: mongo
+    restart: always
+    ports:
+      - 27017:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
```

Comment:
Maybe you can set the username and password from .env?

Reply:
[This commit](https://github.com/aiyu666/whoTalk/pull/23/commits/bfb742546e0dde3ce8123446127356702043aad4) fix